frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json

```diff
@@ -0,0 +1,152 @@
+{
+  "Add": "Add",
+  "Pending Changes": "Pending Changes",
+  "The changes you've made required this virtual machine to be restarted.": "The changes you've made required this virtual machine to be restarted.",
+  "If you make changes to the following settings you will need to restart the virtual machine in order for them to be applied": "If you make changes to the following settings you will need to restart the virtual machine in order for them to be applied",
+  "Restart required to apply changes": "Restart required to apply changes",
+  "Please select a boot source": "Please select a boot source",
```

Comment:
In docs we avoid "please". I looked at Patternfly docs but I didn't see anything. I would update to just say 
"Select a boot source"

Reply:
Fixed!